index.js

```diff
@@ -1,13 +1,36 @@
 /***** Deliverable 1 *****/
-const header = document.querySelector("#header")
+const header = document.querySelector("h1#header") 
 console.log("Here's your header:", header)
 
 
 /***** Deliverable 2 *****/
 
+header.style.color = 'red'
 
 /***** Deliverable 3 *****/
-// console.log("PLAYERS array looks like this:", PLAYERS)
+console.log("PLAYERS array looks like this:", PLAYERS)
 
 
-/***** Deliverable 4 *****/
\ No newline at end of file
+
+var parentElm = document.querySelector(".player-container")
+
+PLAYERS.forEach((player) => {
+    let divPlayer = document.createElement('div')
+    divPlayer.innerHTML = `
+                    <div class='player' data-number="${player.number}"> 
+                    <h3>
+                        ${player.name} (<em>${player.nickname}</em>)
+                    </h3>
+                    <img width = 300px src=${player.photo} alt="${player.name}"/>
+                    </div>
+                    `
+    parentElm.appendChild(divPlayer)
```

Comment:
This will work, but you'll end up creating an extra div for each player, with another div inside it. To get an element that matches the deliverables, you can update this code like this:

```js
   let divPlayer = document.createElement('div')
   divPlayer.classList.add("player")
   divPlayer.dataset.number = player.number
    divPlayer.innerHTML = `
                    <h3>
                        ${player.name} (<em>${player.nickname}</em>)
                    </h3>
                    <img width = 300px src=${player.photo} alt="${player.name}"/>
                    `
    parentElm.appendChild(divPlayer)
```

Check out the Elements tab in the Chrome dev tools and try running this code side-by-side with the code you had originally, and see the difference in the elements that are create!